doc/conf.py

```diff
@@ -1,8 +1,4 @@
-# Configuration file for the Sphinx documentation builder.
-#
-# This file only contains a selection of the most common options. For a full
-# list see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
+from sphinx_mdolab_theme.config import *
```

Comment:
I'm surprised flake8 isn't complaining about this. I do think this is an appropriate situation to use a star import though.

Reply:
The reason it did not complain is because I have manually excluded it: https://github.com/mdolab/pyoptsparse/blob/master/.flake8#L11

Right now it is using the custom `.flake8` file, maybe I should add these settings to the shared repo?

Reply:
I think it makes sense to add it to the shared repo.